packages/pangea-node-sdk/src/types.ts

```diff
@@ -1284,8 +1310,32 @@ export namespace AuthN {
       active_token?: SessionToken;
     }
 
+    export interface ListFilter {
+      active_token_id?: string;
+      active_token_id__contains?: string[];
+      created_at?: string;
+      created_at__gt?: string;
+      created_at__gte?: string;
+      created_at__lt?: string;
+      created_at__lte?: string;
+      email?: string;
+      email__contains?: string[];
+      expire?: string;
+      expire__gt?: string;
+      expire__gte?: string;
+      expire__lt?: string;
+      expire__lte?: string;
+      id?: string;
+      id__contains?: string[];
```

Comment:
I think this is just an issue where we have to update the generated schema as well, but `id__in` also is supports, I believe with all these fields that have __contains